internal/builders/docker/testdata/build-definition.json

```diff
@@ -16,6 +16,8 @@
             }
         },
         "values": {
+            "artifactPath": "config.toml",
+            "command": "cp, internal/builders/docker/testdata/config.toml, config.toml",
```

Comment:
I am not sure if this is how we want the command to be included in the BuildDefinition. Ideally, the command should be an array, but the format only allows a string here.

A better encoding would be `"\"cp\", \"internal/builders/docker/testdata/config.toml\", \"config.toml\""` but I am not sure what is the conventional way to do it in Golang. @asraa Any ideas?

Reply:
Modified it to a JSON encoded array. This should generally work better.

Reply:
Issue filed: https://github.com/slsa-framework/slsa-github-generator/issues/1566